CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Main
 
-* [6099](https://github.com/grafana/loki/pull/6099/files) **cstyan**: Drop lines with malformed JSON in Promtail JSON pipeline stage
+* [6361](https://github.com/grafana/loki/pull/6361) **chaudum**: Add new unwrapped range aggregation `rate_counter()` to LogQL
```

Comment:
do you mind moving these PR entries to `Unreleased/All Changes/Loki/Enhancements (and Changes)`? At some point we started to add them here in the main section but it is wrong I think.